packages/filetree/src/index.js

```diff
@@ -206,7 +211,7 @@ const FileTree = forwardRef(({ projectManager, onSelect, move, copy, initialPath
       return (
         treeNodeContextMenu.map((item, index) => item
           ? <Item key={item.text}
-            onClick={handleMenuItemClick(item)}>{item.text}</Item>
+            onClick={handleMenuItemClick(item)}>{item.bilingualText}</Item>
```

Comment:
please double confirm if there should modify the property

Reply:
change to bilingual so replace the attribute here